coralogix-policies/coralogix-metrics-integration-policy/template.yaml

```diff
@@ -1,10 +1,11 @@
 AWSTemplateFormatVersion: 2010-09-09
 Description: The module will create a role with an inline policy to allow Coralogix to collect metrics from several resource types.
 Parameters:
-  ExternalId:
-    Description: "ExternalId for sts:AssumeRole"
+  ExternalIdSecret:
+    Description: "ExternalIdSecret for sts:AssumeRole"
     Type: "String"
-  AWSAccount:
+    AllowedPattern: "[\\w+=,.:\\/-]*"
+  CoralogixAccount:
```

Comment:
Why is this called `CoralogixAccount`?

Reply:
Should it be Env?

Reply:
`AWSAccount` does not make any sense IMO

Reply:
How would you name these? `[US1, US2, EU1, EU2, AP1, AP2, AP3, dev, staging, custom]`

Reply:
Making it `CoralogixRegion` => https://coralogix.com/docs/user-guides/account-management/account-settings/coralogix-domain/

Reply:
I see plenty of uses of `CoralogixRegion`:

https://github.com/search?q=repo%3Acoralogix%2Fcloudformation-coralogix-aws%20CoralogixRegion&type=code